src/NzbDrone.Core/Indexers/Definitions/AnimeBytes.cs

```diff
@@ -235,300 +233,302 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
 
             var response = JsonConvert.DeserializeObject<AnimeBytesResponse>(indexerResponse.Content);
 
-            if (response.Matches > 0)
+            if (response.Matches == 0)
```

Comment:
I do not think this is even necessary, `Groups` object is always present and in case of 0 matches it will simply be empty. Doing `foreach` on it should be enough to cover all cases without special checks.

Reply:
> This sadly breaks the correct functionality in Prowlarr and Jackett, since it's assuming first is Japanese title, second Romaji title and third Alternative title.

This was always a wrong assumption to make though because we often put multiple titles delimited by `,` in `Alternate Title` field (legacy format doesn't really allow for unlimited number of aliases so that is somewhat of a nasty workaround, but works for search which is what we care about).

Reply:
Mhm. Am I correct in assuming that these values are then used to send back to whoever asked Prowlarr/Jackett for data and not used when searching for data? If so I think the only solution here would be to revert the fix and deal with broken behavior as that is what API consumers been doing so far.

Reply:
I've introduced new field `SynonymnsV2` and reveres old `Synonymns` to previous behavior.

```json
"SynonymnsV2": {
  "Japanese": "はじらいブレイク",
  "Romaji": "",
  "Alternative": "Bashful Break"
}
```